catroid/src/main/java/org/catrobat/catroid/content/Look.java

```diff
@@ -176,7 +176,7 @@ public void createBrightnessContrastHueShader() {
 
 	@Override
 	public void draw(Batch batch, float parentAlpha) {
-		checkImageChanged();
+		checkImageChanged(batch);
```

Comment:
Looking good, but the bigger problem here is that the Look only needs to get updated when LookData/Hue/Brightness is changed, which means it is only necessary to update at the moment they get changed. The imageChanged/hueChanged/brightnessChanged booleans are not threadsave which leads to issues using them in draw(). By only updating Look when setting a new image/hue/brightness it is not needed to check on every draw.

I had to change Look.java for JIRA-237 where I applied the before
mentioned changes. I found out that the refactoring of Look.java fixed this flashing screen bug as well.
I would therefore suggest to close this pr.